### PR TITLE
Add layout breadcrumbs.

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-24713

### Changes
- add breadcrumbs component to the default layout

Peek 2023-03-31 10-31

### CODE REVIEW

File: src/hooks/useBreadcrumbsLinks.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    if (activeNavSegment && isNavItems(activeNavSegment)) {
+      const activeNavigation = extractNavItemGroups(activeNavSegment);
+      const activeFragment = activeNavigation.find((item) => item?.active);
+      if (activeFragment && isExpandableNav(activeFragment)) {
+        const leafFragment = activeFragment.routes.find((item) => item.active);
+        const groupLink = (() => {
+          const appFragments = leafFragment?.href?.split('/');
+          appFragments?.pop();
+          return appFragments ? `${appFragments.join('/')}` : `/${bundleId}`;
+        })();
```
Comment (karelhala) [inline src/hooks/useBreadcrumbsLinks.ts:51]: Any reason to wrap this in IIFE?

Comment (Hyperkid123) [reply src/hooks/useBreadcrumbsLinks.ts:51]: I just felt like it tbh. I can replace it if you'd like.

Comment (karelhala) [reply src/hooks/useBreadcrumbsLinks.ts:51]: Yeah, would be good to change it, no need for extra complexity.

Comment (karelhala) [inline src/hooks/useBreadcrumbsLinks.ts:51]:
```suggestion
        const appFragments = leafFragment?.href?.split('/');
        appFragments?.pop();
        const groupLink = appFragments ? `${appFragments.join('/')}` : `/${bundleId}`;
```

**Review by karelhala — APPROVED** [review]

**Pull request merged**